home-data-migration/src/main/resources/db/changelog/0.0.1/Issue127-QuartzConfig.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="Issue127.001" author="Dandria1802">
```

Comment:
Can we reuse the one that we can take from classpath? e.g. add it into the master changelog:
`    <include file="classpath:org/quartz/impl/jdbcjobstore/liquibase.quartz.init.xml"/>`

Reply:
done